streaming/README.md

```diff
@@ -1,5 +1,150 @@
 # Ray Streaming
 
+Ray Streaming is a streaming data processing framework built on ray. It helps your to build a job
+deals with real-time data easily.
+
+## Key Features
+
+1. **Cross Language**. Based on Ray's multi-language actor, Ray Streaming can also run in multiple
+languages(only Python and Java is supported currently). You can implement your operator in different
+languages and run them in one job.
+
+2. **Single Node Failover**. We designed a special failover mechanism that only needs to rollback the
+failed node it's own, in most cases, to recover the job. This will be a huge benefit if your job is
+sensitive about failure recovery time. In other frameworks like Flink, instead, the entire job should
+be restarted once a node has failure.
+
+## Examples
+
+### Python
+
+```Python
+import ray
+from ray.streaming import StreamingContext
+
+ctx = StreamingContext.Builder() \
+    .build()
+ctx.read_text_file(__file__) \
+    .set_parallelism(1) \
+    .flat_map(lambda x: x.split()) \
+    .map(lambda x: (x, 1)) \
+    .key_by(lambda x: x[0]) \
+    .reduce(lambda old_value, new_value:
+            (old_value[0], old_value[1] + new_value[1])) \
+    .filter(lambda x: "ray" not in x) \
+    .sink(lambda x: print("result", x))
+ctx.submit("word_count")
+```
+
+### Java
+
+```Java
```

Comment:
A runnable class will be great.

Reply:
I have thought about adding `public static void main(xxxx)` and all `import xxx`. But it seems too long to show our API.

As an example for users to have a quick look at our API, I think that's enough. If a user wants to run some codes, he should go to the API reference page, and a more detailed tutorial will be added there.